package.json

```diff
@@ -25,7 +25,7 @@
     "node": true
   },
   "scripts": {
-    "test": "jshint lib && tape test/parse.test.js",
+    "test": "jshint lib && tape test/parse.test.js && tape test/clip.test.js",
```

Comment:
could this be `tape test/*.test.js` so that assertions in the test suites are combined?